Tests/DependencyInjection/DoctrineExtensionTest.php

```diff
@@ -995,6 +1002,13 @@ public function testShardManager(): void
 
         $this->assertTrue($container->hasDefinition('doctrine.dbal.foo_shard_manager'));
         $this->assertFalse($container->hasDefinition('doctrine.dbal.bar_shard_manager'));
+        $this->assertTrue($container->hasDefinition('doctrine.dbal.baz_shard_manager'));
+
+        $fooManagerDef = $container->getDefinition('doctrine.dbal.foo_shard_manager');
+        $bazManagerDef = $container->getDefinition('doctrine.dbal.baz_shard_manager');
+
+        $this->assertEquals('Doctrine\\DBAL\\Sharding\\PoolingShardManager', $fooManagerDef->getClass());
```

Comment:
PoolingShardManager::class. Not sure why CS didn't complain.

Reply:
Fixed